fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resources/ModResourcePackUtil.java

```diff
@@ -35,47 +34,55 @@
 	public static final int PACK_FORMAT_VERSION = 4;
 
 	private ModResourcePackUtil() {
-
 	}
 
-	public static void appendModResourcePacks(List<ResourcePack> packList, ResourceType type) {
+	public static void appendModResourcePacks(List<? super ModNioResourcePack> packList, ResourceType type) {
 		for (ModContainer container : FabricLoader.getInstance().getAllMods()) {
-			if(container.getMetadata().getType().equals("builtin")){
+			if (container.getMetadata().getType().equals("builtin")) {
 				continue;
 			}
-			Path path = container.getRootPath();
-			ResourcePack pack = new ModNioResourcePack(container.getMetadata(), path, null);
+			ModNioResourcePack pack = new ModNioResourcePack(container, null);
 			if (!pack.getNamespaces(type).isEmpty()) {
 				packList.add(pack);
 			}
 		}
 	}
 
-	public static boolean containsDefault(ModMetadata info, String filename) {
-		return "pack.mcmeta".equals(filename);
-	}
-
-	public static InputStream openDefault(ModMetadata info, String filename) {
-		switch (filename) {
-			case "pack.mcmeta":
-				String description = info.getName();
-				if (description == null) {
-					description = "";
-				} else {
-					description = description.replaceAll("\"", "\\\"");
-				}
-				String pack = String.format("{\"pack\":{\"pack_format\":" + PACK_FORMAT_VERSION + ",\"description\":\"%s\"}}", description);
-				return IOUtils.toInputStream(pack, Charsets.UTF_8);
-			default:
-				return null;
-		}
-	}
-
 	public static String getName(ModMetadata info) {
 		if (info.getName() != null) {
 			return info.getName();
 		} else {
 			return "Fabric Mod \"" + info.getId() + "\"";
 		}
 	}
+	
+	static boolean requestsStandalonePackProfile(ModMetadata meta) {
+		CustomValue value = meta.getCustomValue("fabric-resource-loader:requestStandaloneProfile");
```

Comment:
I think `requestStandalonePack` would be clearer.

Reply:
the value has lower camel case after `:` and the fake mod id before `:`